package.json

```diff
@@ -2320,12 +2330,32 @@
 				{
 					"command": "notification.chatSummarizeNotification",
 					"group": "inline@1",
-					"when": "view == notifications:github && (viewItem == 'Issue' || viewItem == 'PullRequest')"
+					"when": "view == notifications:github && (viewItem == 'Issue' || viewItem == 'PullRequest') && config.githubPullRequests.experimental.notificationsView"
+				},
+				{
+					"command": "notification.chatSummarizeNotification",
+					"group": "issues_0@0",
```

Comment:
nit: the name of this group could be `notifications_0` instead of `issues_0`

Reply:
Ah right okay, I thought the word 'issues' is important in defining the right click context menu. Thanks for the feedback!